src/fal/cli/flow_runner.py

```diff
@@ -102,31 +102,47 @@ def _run_sub_graph(
                     for path in mnode.model.get_post_hook_paths()
                 ]
                 post_hooks.extend(model_hooks)
-        if len(fal_nodes) != 0:
-            python_node_scripts = _id_to_fal_scripts(node_graph, fal_dbt, fal_nodes)
-            results = run_scripts(python_node_scripts, fal_dbt)
-            raise_for_run_results_failures(python_node_scripts, results)
 
         if post_hooks:
             results = run_scripts(post_hooks, fal_dbt)
             raise_for_run_results_failures(post_hooks, results)
 
-    if len(after_scripts) != 0:
+        raise_for_dbt_run_errors(output)
+
+        if fal_nodes:
```

Comment:
This cannot be moved after post_hooks. This is where we run Python models.

Probably need to do the `_mark_dbt_nodes_status` again.